### PR TITLE
Change password to password field for jira edit

### PR DESCRIPTION
**Pull request by mikeastock**

### CODE REVIEW

File: app/services/jira/client.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    @password = auth_hash[:password]
+  def initialize(client_root:, username:, password:)
+    @client_root = client_root
+    @username = username
+    @password = password
   end
 
   def my_issues
-    HTTParty.get(my_issues_url, headers)["issues"]
+    HTTParty.get(my_issues_url, headers).fetch("issues")
```
Comment (mcasper) [inline app/services/jira/client.rb]: Might need `.fetch("issues", {})`

**Comment by mcasper** [comment]
Other than the fetch default, +1

**Comment by mikeastock** [comment]
But don't we want the fetch to blow up? That way we know when the request fails?

**Comment by mcasper** [comment]
I suppose so

**Pull request merged**